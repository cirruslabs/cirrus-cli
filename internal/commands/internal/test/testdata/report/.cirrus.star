def main(ctx):
    print("actual log line")
    return [
        {
            "container": {
                "image": "debian:latest",
            }
        }
    ]
