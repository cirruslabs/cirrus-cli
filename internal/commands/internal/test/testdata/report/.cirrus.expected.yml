task:
  container:
    image: ubuntu:latest
